packages/asset-swapper/src/types.ts

```diff
@@ -124,13 +124,21 @@ export interface ForwarderExtensionContractOpts {
     feeRecipient: string;
 }
 
+export interface AffiliateFee {
+    recipient: string;
+    buyTokenFeeAmount: BigNumber;
```

Comment:
MM wants percentage fees.

Reply:
Or I guess you're gonna do the conversion in API?

Reply:
I'm handling the percentages from the 0x-api side